showcase/src/app/demo/toast/demo-toast-notification.component.ts

```diff
@@ -0,0 +1,116 @@
+import { Component, Inject } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {
+	STARK_TOAST_NOTIFICATION_SERVICE,
+	StarkToastNotificationService,
+	StarkToastNotificationResult,
+	StarkToastMessage,
+	StarkMessageType
+} from "@nationalbankbelgium/stark-ui";
+
+@Component({
+	selector: "showcase-demo-toast-notification",
+	styleUrls: ["./demo-toast-notification.component.scss"],
+	templateUrl: "./demo-toast-notification.component.html"
+})
+/**
+ * Component to demo the toast notifications
+ */
+export class DemoToastComponent {
+	/**
+	 * Message, or ID of message displayed in the message in the message box (translations applies).
+	 */
+	public messageText: string = "SHOWCASE.DEMO.TOAST.MESSAGE_INTRO";
+
+	/**
+	 * Message, or ID of message displayed in the toast (translations applies).
+	 */
+	public exampleMessage: string = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus.";
```

Comment:
Can you use a translation key here? Otherwise we are not showing the major feature of this component. Can you also make use of the interpolate values? You can have a look at the example in the old Stark...